docs/reference/changelog.rst

```diff
@@ -13,6 +13,8 @@ New features:
 Bug fixes:
 
 - Correctly handle ``source-subdir`` values on the ``go-use`` plugin.
+- Add the :ref:`jlink plugin<craft_parts_jlink_plugin>` for setting up
+  Java runtime.
```

Comment:
Reminder when this is approved and ready to land: put this in a new entry.

(There's not much value in doing it right now, because it's likely a new changelog entry will be created by another PR)

Reply:
Oh, yes, I missed this. Absolutely, this entry shouldn't be a bug fix.